### ISSUE TITLE
Remove (or change) the dwarven language when there will be more civilisations.

### ISSUE THREAD

**Issue by LordCerapter** [LordCerapter/D4FS#66]

**Comment by Gwolfski** [comment]
Will change it to use for my future civ

**Comment by LordCerapter** [comment]
There is also still the question of #9. Do we bring in new words or what?
Because if we do, we'll need to continuously keep the language files updated.

**Comment by Bearskie** [comment]
Would suggest that the civs creator generates their own languages using DFLang.

**Comment by LordCerapter** [comment]
Yep, I always wanted to do that, but you gotta get people to agree on that.
